### PR TITLE
Beautiful soup tests

### LINKED ISSUE #9: Create Test web scraping urls
## Task Description

- Figure out different avenues in order to scrape the urls from the court website

## User Stories

- As an engineer, I need to plan and research different ideas and ways to propose the best solution, so that it is most effective for my client

## Relevant Files [If Available] 

-

### LINKED ISSUE #87: Add tests for pdf/title scraping
## Task Description

- Functions for getting pdf url and title of case now take a BeautifulSoup object as an input and output strings. More easily tested now.

## User Stories

- As a developer, I need to test to make sure my functions are returning the correct output, so that I know my code works as expected.

## Relevant Files [If Available] 

- `test_extract.py`

### LINKED ISSUE #107: Edit transform script so judge names are returned as all caps.
## Task Description

Edit the function `strip_titles()` to make sure it returns the name in all upper case.

## User Stories

As an engineer I need consistency in the database so it's easier to query and match records.

## Relevant Files 

pipeline/transform.py

### PR DESCRIPTION
**Pull request by luciegamby-sigmalabs**
Adding more tests for extract.py - specifically when dealing with Beautiful Soup.

Closes #87 
Closes #9
Closes #107

### CODE REVIEW

**Comment by luciegamby-sigmalabs** [comment]
Also within this PR, I've changed the transform script so that when cleaned, judge names are all uppercase.

File: pipeline/test_extract.py [hunk 1]
```diff
@@ -51,8 +57,88 @@ def test_combine_case_url_returns_correct_url():
...
     assert first == 1
     assert second == 2
+
+
+"""
+Testing get_case_pdf_url
+"""
+
+
+def test_get_case_pdf_url_returns_string():
```
Comment (ErvinRex) [inline pipeline/test_extract.py:73]: important test, well constructed!

File: pipeline/test_extract.py [hunk 2]
```diff
@@ -51,8 +57,88 @@ def test_combine_case_url_returns_correct_url():
...
+            <h1 class="judgment-toolbar__title">Foobar</h1>
+        """
+
+    fake_soup = BeautifulSoup(html_string, "html.parser")
+    title = get_case_title(fake_soup)
+
+    assert title == "Foobar"
+
+
+def test_get_case_title_changes_slash_to_dash():
```
Comment (ErvinRex) [inline pipeline/test_extract.py:134]: why was it necessary to test this?

**Review by ErvinRex — APPROVED** [review]
Great tests that cover the basic use case of scraping urls and getting necessary information. Well done!

**Pull request merged**